subprojects/instant-execution/src/integTest/groovy/org/gradle/instantexecution/InstantExecutionCompositeBuildsIntegrationTest.groovy

```diff
@@ -19,21 +19,23 @@ package org.gradle.instantexecution
 
 class InstantExecutionCompositeBuildsIntegrationTest extends AbstractInstantExecutionIntegrationTest {
 
-    def "warns for not implemented included builds support"() {
+    def "problem when included builds are present"() {
 
         given:
         def instantExecution = newInstantExecutionFixture()
         settingsFile << """includeBuild("included")"""
         file("included/settings.gradle") << ""
 
         when:
+        withDoNotFailOnProblems()
         instantRun("help")
 
         then:
         instantExecution.assertStateStored()
-        output.contains("Gradle runtime: support for included builds is not yet implemented with instant execution.")
+        expectInstantExecutionProblems("- Gradle runtime: support for included builds is not yet implemented with instant execution.")
```

Comment:
I probably wouldn't include the formatting (the `- `) in the expected problem message.

Reply:
Yep, this will be addressed in a follow up PR.